src/_rules/semantic.js

```diff
@@ -15,4 +15,5 @@ export const semanticRules = [
   [/^s-border-([lrtbxy])-(.+)$/, ([, direction, cssvar]) => directionMap[direction]?.map(
     (dir) => [`border${dir}-color`, h.semanticToken(`border-${cssvar}`)],
   )],
+  [/^s-outline-(.+)$/, ([, cssvar]) => ({ 'outline-color': h.semanticToken(`border-${cssvar}`) })],
```

Comment:
Do we also need this:
`[/^s-outline$/, () => ({ 'outline-color': h.semanticToken('border') })]`
to be able to set the color to `s-border`? 🤔

Reply:
Ah, you might be right! I'll just double-check with @Skadefryd.

Reply:
Yes I guess :) Im not 100% sure I understand the comment but I think we should follow the same patterns as much as possible (when it makes sense)   Outline and border should be mostly the same from the consumers standpoint.

Reply:
Added support for `s-outline` in [e5c9331](https://github.com/warp-ds/drive/pull/173/commits/e5c9331e3347f5300d1dbcfa97a7dec7907c9cbc)